bc_obps/service/contact_service.py

```diff
@@ -91,12 +92,15 @@ def update_contact(
             raise Exception(UNAUTHORIZED_MESSAGE)
 
         address_data = payload.dict(include={'street_address', 'municipality', 'province', 'postal_code'})
-
+        contact_data: Dict = payload.dict(include={*ContactIn.Meta.fields})
```

Comment:
We shouldn't make the service depend on the API layer (the `ContactIn` schema). The service should declare exactly which fields it needs here instead:
```py
payload.dict(include=['first_name', 'last_name', 'email', 'phone_number', 'position_title'])
```
(this solution is already iffy since the type of the payload is an API type instead of a Service layer type)